src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,45 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+public class HomePage extends BasePage{
+        public HomePage(WebDriver givenDriver){
+        super(givenDriver);
+    }
+    By myPlayList = By.xpath("//*[@id=\"playlists\"]/ul/li[3]/a");
+//    By clickEdit = By.xpath("//*[@id=\"playlists\"]/ul/li[3]/nav/ul/li[1]");
+    By textField = By.cssSelector(" [name = 'name']");
+//    By newName = By.cssSelector(" [name = 'name']");
+    By playlistRenamedMsg = By.cssSelector("div.success.show");
+    public void chosePlaylistToRename()  {
+        doubleClick(myPlayList);
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```